packages/wrangler/src/event-subscriptions/index.ts

```diff
@@ -0,0 +1,323 @@
+import { readConfig } from "../config";
+import { createCommand, createNamespace } from "../core/create-command";
+import { CommandLineArgsError } from "../errors";
+import { logger } from "../logger";
+import { requireAuth } from "../user";
+import { PRODUCTS } from "./products";
+
+type EventSubscription = {
+	id: string;
+	created_at: string;
+	modified_at: string;
+	name: string;
+	enabled: boolean;
+	product_id: string;
+	resource_id: string;
+	queue_name: string;
+	events: string[];
+};
+
+type EventSubscriptionEventSpec = {
+	id: string;
+	events: Array<{ name: string; description: string }>;
+};
+
+export const eventSubscriptionsNamespace = createNamespace({
```

Comment:
should this command name be post-fixed with `Command` as well, just like all other commands?
```suggestion
export const eventSubscriptionsNamespaceCommand = createNamespace({
```

Reply:
oh...nvm. Totally missed you're calling `createNamespace`. Ignore the comment